src/include/sof/alloc.h

```diff
@@ -74,7 +74,7 @@ struct block_map {
 	uint16_t first_free;	/* index of first free block */
 	struct block_hdr *block;	/* base block header */
 	uint32_t base;		/* base address of space */
-} __attribute__ ((packed));
+} PLATFORM_DCACHE_ALIGN;
```

Comment:
It's better to use __atribute__ here directly as it make it more obvious. i.e. 
`__attribute__((__aligned__(DCACHE_LINE_SIZE)))`

Reply:
@lgirdwood I agree, but then it won't compile for platforms with DCACHE_LINE_SIZE=0. I decided, that such define would be better than declaring those structs separately for different platforms or even #ifdefing them.